bloxid/v0.go

```diff
@@ -82,14 +103,23 @@ func parseV0(bloxid string) (*V0, error) {
 	}
 
 	switch {
+	case strings.HasPrefix(string(decoded), hashIDPrefix):
```

Comment:
optimization - define `hashIDPrefixBytes` and use bytes.HasPrefix() - see extrinsicIDPrefixBytes example
```suggestion
	case bytes.HasPrefix(decoded, hashIDPrefixBytes):
```